src/views/options/components/command-picker/layout.css

```
@@ -0,0 +1,161 @@
+
+  #filterWrapper {
+    display: flex;
+    flex-direction: row;
+    border-color: var(--border-color);
+    border-width: 1px;
+    border-bottom-style: solid;
+
+    #filterInput {
+      min-width: 0;
+      flex: 1;
+      min-width: 3rem;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)